packaging/testing/smoke/container/fluent-bit.yaml

```diff
@@ -0,0 +1,11 @@
+service:
```

Comment:
can you put a link/document to the implementing PR of YAML format or a doc?

Reply:
done